Configuration.props

```diff
@@ -6,26 +6,40 @@
   <Import
       Project="$(MSBuildThisFileDirectory)Configuration.Override.props"
       Condition="Exists('$(MSBuildThisFileDirectory)Configuration.Override.props')"
-  />
+      />
+
+  <!--
+      Note: this file *must* be imported *after* Configuration.Override.props
+  -->
+  <Import Project="$(MSBuildThisFileDirectory)Configuration.OperatingSystem.props" />
   <PropertyGroup>
     <HostOS Condition=" '$(HostOS)' == '' And '$(OS)' == 'Windows_NT' ">Windows</HostOS>
-    <HostOS Condition=" '$(HostOS)' == '' And '$(OS)' == 'Unix' And Exists ('/Applications') ">Darwin</HostOS>
-    <HostOS Condition=" '$(HostOS)' == '' And '$(OS)' == 'Unix' ">Linux</HostOS>
+    <HostCc Condition=" '$(HostCc)' == '' ">$(HostCc64)</HostCc>
+    <HostCxx Condition=" '$(HostCxx)' == '' ">$(HostCxx64)</HostCxx>
+    <HostCc Condition=" '$(HostCc)' == '' ">$(HostCc32)</HostCc>
+    <HostCxx Condition=" '$(HostCxx)' == '' ">$(HostCxx32)</HostCxx>
     <HostCc Condition=" '$(HostCc)' == '' ">cc</HostCc>
     <HostCxx Condition=" '$(HostCxx)' == '' ">c++</HostCxx>
+    <HostCcName Condition=" '$(HostCcName)' == '' ">cc</HostCcName>
```

Comment:
I don't understand why `$(HostCcName)` exists.

The only place it's used is in a `@(RequiredProgram)`, but the whole point to `@(RequiredProgram)` is to ensure that the program exists within `$PATH`, so dropping `<RequiredProgram Include="$(HostCc)" />` and replacing with `<RequiredProgram Include="$(HostCcName)" />` makes no sense at all to me.

The only thing that makes any plausible sense is that `$(HostCcName)` is the *name of the package that contains `$(HostCc)`*, which *might* make sense, but if that's the intention then `%(RequiredProgram.Homebrew)` should be used instead:

    <!-- in android-toolchain.projitems -->
    <RequiredProgram Include="$(HostCc)">
      <Homebrew>$(HostCcName)</Homebrew>
    </RequiredProgram>

If this is the intended use case, then `$(HostCcName)` should be named `$(HostCcPackage)`.